blog/_posts/2018-07-08-4Ps-of-GirlsCode-MK.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Team GirlsCode MK - The 4 Ps of GirlsCode MK
+layout: post
+created_at: Sun Jul 08 2018
+permalink: blog/_posts/2018-07-08-4ps-of-girlscode-mk
```

Comment:
Here, permalink need to be same as file name. Please check the old blog posts for an example.